src/main/scala/viper/server/frontends/lsp/DataProtocol.scala

```diff
@@ -107,7 +108,7 @@ case class PlatformDependentURL (
               linux: Option[String])
 
 // scope == null means global scope
-case class Definition(definition_type: String, name: String, code_location: Position, scope: Range)
+case class Definition(definition_type: SymbolType, name: String, code_location: Position, scope: Range)
```

Comment:
Is there a particular reason to use a different parameter order compared to Silver resp. redeclaring this class:
```
case class Definition(name: String, typ: SymbolType, location: viper.silver.ast.Position,
                         scope: Option[viper.silver.ast.AbstractSourcePosition] = None)
```

Reply:
No reason. I don't know why the student who worked on LSP decided to duplicate this class. I'll add a comment that explains that this should probably be refactored.